components/brave_extension/extension/brave_extension/background/events.ts

```diff
@@ -7,7 +7,6 @@ require('./events/tabsEvents')
 require('./events/shieldsEvents')
 require('./events/runtimeEvents')
 require('./events/webNavigationEvents')
-require('./events/cosmeticFilterEvents')
```

Comment:
This is still required, see my comment on it below.

Reply:
added back 😄